### ISSUE TITLE
Icons are showing as question marks "???"

### ISSUE THREAD

**Issue by NielZeeman** [OfficeDev/office-ui-fabric-react#4329]
### Bug Report

- __Package version(s)__: office-ui-fabric-react@5.65.0
- __Browser and OS versions__: Windows 10 : Edge (41.16299.248.0), Chrome (65.0.3325.181) & FireFox (59.0.1)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority:  Normal

Products/sites affected: Visual Studio Team Services (VSTS)

#### Describe the issue:

I have created a VSTS extension using React/Typescript with the Office UI Fabric React library. Running it on an on-prem instance of TFS everything renders fine, but publishing it into VSTS the icons on the commandbar renders as question marks.

#### Actual behavior:

Icons render as question marks when publishing it to VSTS, but renders fine on-prem.
![image](https://user-images.githubusercontent.com/16056816/37698368-40288812-2ceb-11e8-90a4-b6039f1a171d.png)
#### Expected behavior:

Icons to render the same on-prem and in VSTS
![ouif_workingicons](https://user-images.githubusercontent.com/16056816/37698608-5331e222-2cec-11e8-84ea-0679602e3cdd.PNG)

**Comment by dzearing** [comment]
@cschleiden this seems like `initializeIcons` is not being called correctly in this case, maybe?

Can you help?

**Comment by dzearing** [comment]
@NielZeeman I highly suspect that a CSS rule is causing the issue. Can you inspect the element and look at the `font-family`? Normally "missing" icons show as boxes. When you see question marks, it's rendering the correct unicode values as a text based font instead of the icon font.

Something like this would do it!

```css
* {
  font-family: Times New Roman;
}
```

**Comment by cschleiden** [comment]
Oh, didn't see the mention before. I would be surprised, since VSTS also uses fabric icons. 

@NielZeeman I'm using fabric in a number of VSTS extensions, one example: https://github.com/cschleiden/vsts-pr-templates and the icons show up fine. Maybe you can compare your code to mine to see where the difference is.

**Comment by NielZeeman** [comment]
@dzearing The element looks like this 
`<i class="css-40 root-36 ms-CommandBarItem-icon itemIcon_099220ae ms-CommandBarItem-iconColor itemIconColor_099220ae" role="presentation" aria-hidden="true" data-icon-name="Refresh">???</i>`
And the computed font is:
`font-family: "FabricMDL2Icons-0";`
Chrome seems to have the unicode char instead of the question marks.

@cschleiden Thanks, I have a fabric extension that works, but it is using an older version of fabric. I have not tried to update it to see if it works with the latest. That said, I don't see much different with what you are doing in your extension as compared to mine. The only difference I did find was that you specified iconType as part of the iconProps, but that did not do anything when I updated it.

**Comment by NielZeeman** [comment]
@dzearing  && @cschleiden  Is there anything else I can try ?

**Comment by cschleiden** [comment]
How do you import fabric? You don't get our version by accident?

**Comment by NielZeeman** [comment]
Hi @cschleiden 
I import explicitly:
`"dependencies": {
    "@types/q": "^1.5.0",
    "applicationinsights-js": "^1.0.14",
    "office-ui-fabric-react": "^5.65.0",
    "q": "^1.5.1",
    "react": "^16.0.0",
    "react-dom": "^16.0.0",
    "telemetryclient-team-services-extension": "^1.0.2",
    "ts-loader": "^3.1.1",
    "vss-web-extension-sdk": "^3.121.0"
  },`

I have restructured the app breaking out the components into smaller bits trying to see if that helps, but nothing.
Next I'm going to start from scratch and build it up on piece at a time. The problem is that it only surfaces this behavior on VSTS :(
I was thinking it may be something to do with CORS, but I have a grid with the persona component that renders fine.

**Comment by NielZeeman** [comment]
I have changed the index to only have this:
```
import * as React from "react";
import * as ReactDOM from "react-dom";

import { initializeIcons } from "@uifabric/icons";
import { Dropdown } from "office-ui-fabric-react/lib/Dropdown";

export function init(containerId: string): void {
    initializeIcons();
    ReactDOM.render(<Dropdown
        placeHolder="Some placeholder"
        style={{ marginTop: "5px", width: 400 }}
    />, document.getElementById(containerId));
}
```
the question marks still show instead of the "down arrow" icon and the rendered HTML looks like this:
```
 <div class="ms-Dropdown-container">
    <div tabindex="0" class="ms-Dropdown root_af68fb60" id="Dropdown0" role="listbox" aria-expanded="false" aria-describedby="Dropdown0-option" aria-live="assertive" style="width: 400px; margin-top: 5px;" aria-autocomplete="none" data-is-focusable="true">
    <span class="ms-Dropdown-title title_af68fb60 ms-Dropdown-titleIsPlaceHolder titleIsPlaceHolder_af68fb60" id="Dropdown0-option" role="listbox" aria-readonly="true" aria-atomic="true">
    <span>Some placeholder</span>
    </span>
    <span class="ms-Dropdown-caretDownWrapper caretDownWrapper_af68fb60">
    <i class="css-40 root-37 ms-Dropdown-caretDown caretDown_af68fb60" role="presentation" aria-hidden="true" data-icon-name="ChevronDown">???</i>
    </span>
    </div>
    </div>
```

**Comment by dzearing** [comment]
@NielZeeman network trace? Do you see any requests to fabricicons* files? Paste the urls.

**Comment by dzearing** [comment]
@NielZeeman any update? Would like to help you if you're still having issues.

**Comment by NielZeeman** [comment]
@dzearing I had to revert to a point before I chucked out the OUIF to get a working extension out. Initial checks seem to have resolved this issue making me think that it was VSTS that had the problem. My code was exactly what it was, with the version of packages that gave me the issue. 

I will spend a bit more time to see if it gives me the same or expected behavior, but for now we can close this issue.
